apps/coscrad-frontend/src/configurable-front-matter/validation/validate-configurable-content.ts

```diff
@@ -6,6 +6,7 @@ import {
 import { configurableContentPropertiesAndConstraints } from '../data/configurable-content-schema';
 
 export const validateConfigurableContent = (instance: unknown): Error[] =>
+    // TODO validate that defaultLanguageCode is a LanguageCode
```

Comment:
If you don't do this now, make a story for this and I can pair with you on this later.